7_drivers_munge/src/GLM_driver_utils.R

```diff
@@ -141,7 +141,7 @@ feathers_to_driver_file <- function(filepath, cell_group_table){
            Rain = apcpsfc*24/1000) %>%
     mutate(Snow = ifelse(AirTemp < 0, Rain*10, 0), Rain = ifelse(AirTemp < 0, 0, Rain)) %>% #convert to m/day rate)
     select(time, ShortWave, LongWave, AirTemp, RelHum, WindSpeed, Rain, Snow) %>% # now downsample?
-    mutate(date = lubridate::as_date(time)) %>% group_by(date) %>%
+    mutate(date = lubridate::as_date(time, tz = "Etc/GMT+6")) %>% group_by(date) %>%
```

Comment:
Is it always appropriate to specify GMT+6? Would this work on my computer in EST? What was the motivation for this change?

Reply:
No, this is a footprint-specific change, making sure the hourly data end up summarized to the proper date. Two of the three states are in CT, so that is why this is used, and only lakes from those two states were in the WRR analysis.

Reply:
Hmm, so maybe now would be the time to make this an if-else or switch statement based on the state? or at least make an issue for it - this is the kind of thing I'll forget now and never remember.

Reply:
Yes, there is an issue for it: https://github.com/USGS-R/lake-temperature-model-prep/issues/27

Reply:
Oh, excellent!